etc/ci/performance/runner.py

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env python3
+
+import argparse
+import itertools
+import json
+import os
+import subprocess
+from statistics import median, StatisticsError
+
+
+def load_manifest(filename):
+    with open(filename, 'r') as f:
+        text = f.read()
+    return list(parse_manifest(text))
+
+
+def parse_manifest(text):
+    return filter(lambda x: x != "" and not x.startswith("#"),
+                  map(lambda x: x.strip(), text.splitlines()))
+
+
+def execute_test(url, command, timeout):
+    print("Running test:")
+    print(command)
+    print("Timeout:{}".format(timeout))
+    try:
+        return subprocess.check_output(command, stderr=subprocess.STDOUT,
+                                       shell=True, timeout=timeout)
+    except subprocess.CalledProcessError as e:
+        print("Unexpected Fail:")
+        print(e)
+        print("You may want to re-run the test manually:\n{}".format(command))
+    except subprocess.TimeoutExpired:
+        print("Test timeout: {}".format(url))
+    return ""
+
+
+def get_servo_command(url, timeout):
+    ua_script_path = "{}/user-agent-js".format(os.getcwd())
+    test_cmd = ("timeout {timeout}s ./servo/servo '{url}'"
```

Comment:
Since this is in-tree, the easiest way to get a servo binary is `./mach build` (`--release` I assume). After that with the PWD being the top level of the servo repo, we can just use `target/release/servo` instead of requiring copying the `servo` binary to `etc/ci/performance/servo/servo`.